db:migrations:001_create_shops.rb

```diff
@@ -0,0 +1,17 @@
+require "sequel"
+
+DB = Sequel.sqlite # connects to sqlite3 using sequel
+
+# create an items table
+DB.create_table :shops do
```

Comment:
@mlan080 this is an example on how to use `sequel`.

in our case, we want to have separate migrations files to practice the schema migration concept used on modern frameworks like rails.

changes will come, so we need a proper way of evolving the database schema :)